initrd/init

```diff
@@ -103,9 +103,13 @@ fi
 
 #Specify whiptail background colors cues under FBWhiptail only
 if [ -x /bin/fbwhiptail ]; then
+	export NEWT_WARNING=heads_fb_enabled=true
+	export NEWT_ERROR=heads_fb_enabled=true
```

Comment:
Does this do anything?  Since we're not using Newt here I don't think anything will care about this

Reply:
It's so the env call doesn't blow up. :) So, does it "do" anything, nope. Just prevents bad things from happening.